### PR TITLE
Added meta.json and logo.png for JSF Form Archive Utility

### PR DESCRIPTION
**Pull request by nqhoan-axonivy**
Dear @ivy-sgi could you help review this PR and give your suggestion on the following:
- the logo for this new utility
- the translation text for name and description
Thank you so much!

### CODE REVIEW

**Comment by nqhoan-axonivy** [comment]
For the logo, it's hard for me 🐤 
I found and just have 2 ideas:
image
image
@ivy-sgi do you have any idea for it?

File: market/utils/jsf-formarchive-util/meta.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "id": "jsf-formarchive-util",
+  "name": "JSF Form Archive Utility",
+  "names": [
+    {
+      "locale": "en",
+      "value": "JSF Form Archive Utility"
+    },
+    {
+      "locale": "de",
+      "value": "TODO JSF Form Archive Utility"
```
Comment (nqhoan-axonivy) [inline market/utils/jsf-formarchive-util/meta.json:12]: I need your help @ivy-sgi

**Comment by ivy-sgi** [comment]
Hi @nqhoan-axonivy ! Why not using the standard AxonIvy Utility Logo already added to this connector?

I would change the name to 
en: UI Archive Utility
de:  UI Archiv Utility 

en: This tool allows you to capture snapshots of your UIs, making it easy to save and track what inputs users made at specific times.
de: Archiviere Screenshots der Benutzeroberflächen und dokumentiere so die Eingaben deiner Nutzer zu bestimmten Zeitpunkten

File: market/utils/jsf-formarchive-util/logo.png
Comment (nqhoan-axonivy) [inline market/utils/jsf-formarchive-util/logo.png:1]: Thank @ivy-sgi so much for your suggestion!
For logo, this standard logo is fine, right?
If yes, could you help approve this PR.

**Comment by nqhoan-axonivy** [comment]
> Hi @nqhoan-axonivy ! Why not using the standard AxonIvy Utility Logo already added to this connector?
> 
> I would change the name to en: UI Archive Utility de: UI Archiv Utility
> 
> en: This tool allows you to capture snapshots of your UIs, making it easy to save and track what inputs users made at specific times. de: Archiviere Screenshots der Benutzeroberflächen und dokumentiere so die Eingaben deiner Nutzer zu bestimmten Zeitpunkten

Ah I just want to find a logo that relates to the feature that this tool supports instead of using the default logo

**Pull request merged**